### PR TITLE
Fixing README and CI

### PR DESCRIPTION
**Pull request by capnkenny**

### CODE REVIEW

**Comment by capnkenny** [comment]
@RubyNova - does this potentially fix #195 ? Since we're providing an exported version now, we shouldn't need to worry much about the main vcpkg repo since any updates will be managed purely by us now.

File: scripts/machine-setup.ps1 [hunk 1]
```diff
@@ -3,10 +3,11 @@
 
 $VcpkgInstallDirectory = Join-Path -Path $HOME -ChildPath "vcpkg"
 $DotNetInstallDirectory = Join-Path -Path $HOME -ChildPath "dotnet"
+$ProgressPreference = 'SilentlyContinue'
 
 if (!(Get-Command python -ErrorAction SilentlyContinue ))
 {
-  $PythonInstaller = New-TemporaryFile | Rename-Item -NewName { $_.Name + ".exe" } - PassThru
```
Comment (RubyNova) [inline scripts/machine-setup.ps1:9]: What actually changed here?

Comment (capnkenny) [reply scripts/machine-setup.ps1:9]: There was a typo that could potentially cause issues in Powershell - not world ending as the command would run, but removing the space between `-` and `PassThru` alleviates the potential issue.

**Review by RubyNova — APPROVED** [review]
LGTM

**Pull request merged**